doc/jaegertracing/index.rst

```diff
@@ -1,91 +1,98 @@
+Refer to one of the following:
 
-`Cephadm Jaeger services deployment <../cephadm/services/tracing/>`_
+`Cephadm Jaeger Services Deployment <../cephadm/services/tracing/>`_
 
-`jaeger deployment <https://www.jaegertracing.io/docs/1.25/deployment/>`_
+`Jaeger Deployment <https://www.jaegertracing.io/docs/1.25/deployment/>`_
 
-`jaeger performance tuning <https://www.jaegertracing.io/docs/1.25/performance-tuning/>`_
+`Jaeger Performance Tuning <https://www.jaegertracing.io/docs/1.25/performance-tuning/>`_
 
 
 Important Notes:
 ^^^^^^^^^^^^^^^^
 
-- Spans are being sent to local jaeger agent, so the jaeger agent must be running on each host (not in all-in-one mode). otherwise, spans of hosts without active jaeger agent will be lost.
+- The Jaeger agent must be running on each host (and not running in all-in-one
+  mode). This is because spans are sent to the local Jaeger agent. Spans of
+  hosts that do not have active Jaeger agents will be lost.
 
-- Ceph tracers are configured to send tracers to agents that are listening to port 6799, so on manual jaeger deployment, option "--processor.jaeger-compact.server-host-port=6799" should be used.
+- Ceph tracers are configured to send tracers to agents that listen to port
+  6799. Use the option "--processor.jaeger-compact.server-host-port=6799" for
+  manual Jaeger deployments. 
 
 
 HOW TO ENABLE TRACING IN CEPH
 -----------------------------
 
-tracing in Ceph is disabled by default.
-it could be enabled globally, or for each entity separately (e.g. rgw).
+Tracing in Ceph is disabled by default.
+
+Tracing can be enabled globally, and tracing can also be enabled separately for
+each entity (for example, for rgw).
+
+Enable tracing globally:
+
+.. prompt:: bash $
 
-  Enable tracing globally::
+   ceph config set global jaeger_tracing_enable true
 
-      $ ceph config set global jaeger_tracing_enable true
 
+Enable tracing for each entity:
 
-  Enable tracing for each entity::
+.. prompt:: bash $
 
-      $ ceph config set <entity> jaeger_tracing_enable true
+   ceph config set <entity> jaeger_tracing_enable true
 
 
 TRACES IN RGW
 -------------
 
-traces of RGW can be found under Service `rgw` in Jaeger Frontend.
+Traces run on RGW can be found under the Service `rgw` in the Jaeger Frontend.
 
 REQUESTS
 ^^^^^^^^
-every user request is being traced. each trace contains tags for
-`Operation name`, `User id`, `Object name` and `Bucket name`.
+Every user request is traced. Each trace contains tags for `Operation name`,
+`User id`, `Object name` and `Bucket name`.
 
-there is also `Upload id` tag for Multipart upload operations.
+There is also an `Upload id` tag for Multipart upload operations.
 
-request trace is named `<command> <transaction id>`.
+The names of request traces have the following format: `<command> <transaction
+id>`.
 
 MULTIPART UPLOAD
 ^^^^^^^^^^^^^^^^
-there is a trace, that consists a span for each request that made by that multipart upload, including all `Put Object` requests.
```

Comment:
@neha-ojha What's a multipart upload?

Reply:
Big RGW objects can be uploaded in peices

Reply:
@anthonyeleven Thanks, Anthony. Good looking out.

Reply:
Given the context here I don't think explaining the term is warranted.